GUI/SearchTaskWindow.xaml

```diff
@@ -309,7 +559,15 @@
                                     </TreeView>
                                 </StackPanel>
                                 <StackPanel Grid.Column="1">
-                                    <Label Content="Variable" />
+                                    <Label Content="Variable"  ToolTipService.ShowDuration="999999" ToolTipService.InitialShowDelay="1000">
+                                        <Label.ToolTip>
+                                            <TextBlock>
+                                                Only use this for highly prevelant modifications (>10% of peptides).
+                                                <LineBreak/>
+                                                All PTMs that are annotated in the database (GPTMD does annotation!) will be automatically searched.
```

Comment:
All PTMs that are annotated in the database (e.g., with G-PTM-D) will be automatically searched. You do not need to check those PTMs here.